packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -4,11 +4,11 @@
 import 'jsdom-global/register';
 import {cloneDeep, forEach, isEqual, isUndefined} from 'lodash';
 import sinon from 'sinon';
-import * as internalMediaModule from '@webex/internal-media-core';
+import * as InternalMediaModule from '@webex/internal-media-core';
```

Comment:
Ah, that's weird that we had both `internalMediaModule` and `InternalMediaCoreModule` previously. I would stick with the `InternalMediaCoreModule` name though, since that is the name of the imported module.

Reply:
@brycetham aligned naming for tests in `plugin-meetings`, `calling` and `media-helpers`

Thanks for that